### ISSUE TITLE
Can't run commands on ECS-Deploy Docker image

### ISSUE THREAD

**Issue by robertomoutinho** [fabfuel/ecs-deploy#149]
```
docker run fabfuel/ecs-deploy:1.7.1 ecs run uat uat-service -c service "rake db:migrate" --securitygroup sg-XXXXXXXX --subnet subnet-XXXXXXXXX --region us-east-2 --access-key-id `echo $AWS_ACCESS_KEY_ID` --secret-access-key `echo $AWS_SECRET_ACCESS_KEY`

Error: no such option: --securitygroup
```

and if I try to run it without the parameters

```
An error occurred (InvalidParameterException) when calling the RunTask operation: Network Configuration must be provided when networkMode 'awsvpc' is specified.
```

**Comment by robertomoutinho** [comment]
The documentation seems to be out of date... version 1.8.0 of the docker image works as expected

**Comment by fabfuel** [comment]
Indeed - good catch, thanks! The future is only available from version 1.8.0. I will update the readme.

Best
Fabian
